src/Jackett.Common/Indexers/TVstore.cs

```diff
@@ -103,95 +104,99 @@ public double UploadFactorCalculator(DateTime dateTime, string type)
+                        .Content;
+                    var firstFileName = filesList.Split(
+                        new[]
+                        {
+                            @"\\"
+                        }, StringSplitOptions.None)[1];
+                    // Delete the file extension. Many first files are either mkv or nfo.
+                    // Cannot confirm these are the only extensions, so generic remove all 3 char extensions at end of section.
+                    firstFileName = Regex.Replace(firstFileName, @"\.\w{3}$", string.Empty);
+                    if (isSeasonPack)
+                        firstFileName = Regex.Replace(
+                            firstFileName, @"(?<=S\d+)E\d{2,3}", string.Empty, RegexOptions.IgnoreCase);
+                    var category = new[]
                     {
-                        release.Title = fileinf[1];
-                        if (fileinf[1].Length > 5 && fileinf[1].Substring(fileinf[1].Length - 4).Contains("."))
-                            release.Title = fileinf[1].Substring(0, fileinf[1].Length - 4);
-                    }
-                    // SeasonPack check
-                    if (parameters[is_season_id].Contains("évad/"))
+                        TvCategoryParser.ParseTvShowQuality(firstFileName)
+                    };
+                    var release = new ReleaseInfo
                     {
-                        type = "season";
-                        // If this is a seasonpack, remove episode nunmber from title.
-                        release.Title = Regex.Replace(release.Title, "s0?(\\d+)(e0?(\\d+))", "S$1", RegexOptions.IgnoreCase);
-                    }
-
-                    release.PublishDate = DateTime.Parse(parameters[publish_date_id], CultureInfo.InvariantCulture);
-                    release.Files = int.Parse(parameters[files_id]);
-                    release.Size = long.Parse(parameters[size_id]);
-                    release.Seeders = int.Parse(parameters[seeders_id]);
-                    release.Peers = (int.Parse(parameters[peers_id]) + release.Seeders);
-                    release.Grabs = int.Parse(parameters[grabs_id]);
-                    release.MinimumRatio = 1;
-                    release.MinimumSeedTime = 172800; // 48 hours
-                    release.DownloadVolumeFactor = 1;
-                    release.UploadVolumeFactor = UploadFactorCalculator(release.PublishDate, type);
-                    release.Category = new List<int> { TvCategoryParser.ParseTvShowQuality(release.Title) };
-                    if ((already_found + releases.Count) < limit)
-                        releases.Add(release);
-                    else
-                        return releases;
+                        Title = firstFileName,
+                        Link = downloadLink,
+                        Guid = downloadLink,
+                        PublishDate = publishDate,
```

Comment:
Do we have comments link? Can it be build with the id or something?

Reply:
There is no details pages on this tracker, it's a turndown menu in the search results page that's generated from a JS call. As far as I can tell, there was no explicit url you can navigate to that brings you to this view. The original code had `Link = Guid = Comments`